src/mono/wasm/debugger/DebuggerTestSuite/BreakpointTests.cs

```diff
@@ -448,5 +448,29 @@ public async Task DebugHotReloadMethodEmpty()
             pause_location = await SendCommandAndCheck(JObject.FromObject(new { }), "Debugger.resume", "dotnet://ApplyUpdateReferencedAssembly.dll/MethodBody1.cs", 38, 8, "StaticMethod4");
             locals = await GetProperties(pause_location["callFrames"][0]["callFrameId"].Value<string>());
         }
+
+        [Fact]
+        public async Task ConditionalBreakpointInALoop()
+        {
+            var bp_conditional = await SetBreakpointInMethod("debugger-test.dll", "LoopClass", "LoopToBreak", 4, condition:"i == 3");
```

Comment:
Another condition to try would be something like `i % 3 == 0`, which would cause the breakpoint to be hit more than once, but on non-subsequent runs.

Reply:
And maybe one that fails to evaluate at all

Reply:
done.